packages/compass-aggregations/src/modules/saved-pipeline.ts

```diff
@@ -86,6 +86,39 @@ export const updatePipelineList =
       });
   };
 
+export const handleNamespaceChanged = ({
```

Comment:
For the follow-up PRs: would it make more sense to do this in rename collection plugin instead of here? You'd also need to do it for saved queries and recent queries and spreading this logic across multiple plugins doesn't feel right